opensearch/src/test/java/org/opensearch/sql/opensearch/storage/script/aggregation/AggregationQueryBuilderTest.java

```diff
@@ -148,6 +151,32 @@ void should_build_type_mapping_for_field_reference() {
         ));
   }
 
+  @Test
+  void should_build_type_mapping_for_datetime_type() {
```

Comment:
Yes, changes were put in place because aggregation IT like [testMinDateTimePushedDown](https://github.com/opensearch-project/sql/blob/a924d5a16bc0f24ef77fb4af5ac7e967cca84a43/integ-test/src/test/java/org/opensearch/sql/sql/AggregationIT.java#LL401C15-L401C40) were failing. Since they now pass, then this case should be covered.